compiler-plugin/src/main/kotlin/com/google/devtools/ksp/KotlinSymbolProcessingExtension.kt

```diff
@@ -509,3 +513,13 @@ fun findTargetInfos(languageVersionSettings: LanguageVersionSettings, module: Mo
             else -> UnknownPlatformInfoImpl(platform.toString())
         }
     } ?: emptyList()
+
+// FIXME: remove as soon as possible.
```

Comment:
What is the plan (on what condition) to remove this wordkaround

Reply:
The fix just landed in the Kotlin compiler and should be available in the next bootstrap release and Kotlin 2.1.0-RC.